README.md

```diff
@@ -49,60 +46,47 @@
 - Use React components.
 - Use React props.
 - Use React state.
+- Use React hooks.
+- Use JavaScript events.
+- Use React life cycle methods.
+- Understand the mechanism of lifting state up.
 
 ---
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-<details>
-  <summary>Markup</summary>
-  <ul>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Dynamic</summary>
-  <ul>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Styles</summary>
-  <ul>
-    <li><a href="https://sass-lang.com/">SASS</a></li>
-    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Frameworks</summary>
   <ul>
-    <li><a href="https://react.dev/">React.js</a></li>
+    <li>
+      <img src="https://skillicons.dev/icons?i=react"/>
+      <a href="https://react.dev/">React.js</a>
+    </li>
+    <li>
+      <img src="https://skillicons.dev/icons?i=js"/>
+      <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a>
+    </li>
+    <li>
+      <img src="https://skillicons.dev/icons?i=sass"/>
+      <a href="https://sass-lang.com/">SASS</a>
+    </li>
+    <li>
+      <img src="https://skillicons.dev/icons?i=bootstrap"/>
+      <a href="https://getbootstrap.com/">Bootstrap</a>
+    </li>
+    <li>
```

Comment:
Great to see you are going to use Bootstrap! Kindly, at this point of your project I suggest you include it in your Future Features; the same as the logo, until you start applying them.

Reply:
✍🏼 Absolutely right, Bootstrap should be on my **Future Features**. noted! thank you Hernan!